supervisor/api/network.py

```diff
@@ -209,20 +209,17 @@ async def interface_update(self, request: web.Request) -> None:
         for key, config in body.items():
             if key == ATTR_IPV4:
                 interface.ipv4setting = replace(
-                    interface.ipv4setting
-                    or IpSetting(InterfaceMethod.STATIC, [], None, []),
+                    IpSetting(InterfaceMethod.STATIC, [], None, []),
                     **config,
                 )
             elif key == ATTR_IPV6:
                 interface.ipv6setting = replace(
-                    interface.ipv6setting
-                    or IpSetting(InterfaceMethod.STATIC, [], None, []),
+                    IpSetting(InterfaceMethod.STATIC, [], None, []),
                     **config,
                 )
             elif key == ATTR_WIFI:
                 interface.wifi = replace(
-                    interface.wifi
-                    or WifiConfig(
+                    WifiConfig(
                         WifiMode.INFRASTRUCTURE, "", AuthMethod.OPEN, None, None
                     ),
                     **config,
```

Comment:
I forget, do we have UI around this or is purely set via CLI? If we have UI then I'd just want to check that all the fields you can set are actually exposed in the UI before okaying this. Unfortunately I'm realizing I don't actually have a wifi-based test device at the moment, I'll have to look into that